thirdparty/nginx/nginx-1.7.1-zimbra/src/mail/ngx_mail_throttle_module.c

```diff
@@ -264,15 +364,15 @@ void ngx_mail_throttle_ip (ngx_str_t ip, throttle_callback_t *callback)
     pool = callback->pool;
     log = callback->log;
 
-    ngx_log_error (NGX_LOG_INFO, log, 0, "check ip throttle:[%V]", &ip);
+    ngx_log_error (NGX_LOG_INFO, log, 0, "check ip throttle:[%ui, %V]", protocol, &ip);
```

Comment:
Gren that's a good idea.  I'll see about updating that after I get a little more feedback.